ofrak_core/Makefile

```diff
@@ -23,6 +23,7 @@ ofrak/gui/public:
 		cp -r /ofrak_gui ofrak/gui/public ; \
 	elif [ -d ../frontend ]; then \
 		cd ../frontend && \
+		npm install && \
```

Comment:
The downside of having it be automatically executed from `frontend/Makefile` is that for docker, we run it in `Dockerstub` instead. I did add a separate target to `frontend/Makefile` that runs `npm install; npm run build` that the install script now runs in a somewhat ad-hoc way, but I also kept these things inside the `ofrak_core/Makefile`. I am not sure why `ofrak_core/Makefile` rather than `frontend/Makefile` is responsible for any of these things to begin with, but would rather mess with things I do not fully understand, if I do not have to.